config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  resources :posts
```

Comment:
Kindly add corresponding resources in your routes file which allows [:new, :create, :index].